python/ray/tests/test_gpu_objects_nixl.py

```diff
@@ -7,10 +7,19 @@
 @ray.remote(num_gpus=1, num_cpus=0, enable_tensor_transport=True)
 class GPUTestActor:
     @ray.method(tensor_transport="nixl")
-    def echo(self, data):
+    def echo_gpu(self, data):
```

Comment:
nit:

We can pass `device` into `echo` and `sum` so that we don't need to define similar functions twice.